arch/wasm32/wasm.js

```diff
@@ -821,66 +825,70 @@ var musl_hack = (function() {
   return {
     abort: NYI('abort'),
     __addtf3: NYI('__addtf3'),
-    __block_all_sigs: NYI('__block_all_sigs'),
+    __clone: NYI('__clone'),
     __divtf3: NYI('__divtf3'),
-    __dl_invalid_handle: NYI('__dl_invalid_handle'),
-    __dl_seterr: NYI('__dl_seterr'),
+    __eqtf2: NYI('__eqtf2'),
+    exit: NYI('exit'),
     _Exit: NYI('_Exit'),
     __extenddftf2: NYI('__extenddftf2'),
     __extendsftf2: NYI('__extendsftf2'),
-    fcntl: NYI('fcntl'),
-    fdopen: NYI('fdopen'),
-    __fdopen: NYI('__fdopen'),
-    feclearexcept: NYI('feclearexcept'),
-    fegetround: NYI('fegetround'),
-    feraiseexcept: NYI('feraiseexcept'),
-    fesetround: NYI('fesetround'),
-    fetestexcept: NYI('fetestexcept'),
     __fixtfdi: NYI('__fixtfdi'),
     __fixtfsi: NYI('__fixtfsi'),
-    __floatscan: NYI('__floatscan'),
+    __fixunstfsi: NYI('__fixunstfsi'),
     __floatsitf: NYI('__floatsitf'),
-    fprintf: NYI('fprintf'),
-    iswctype: NYI('iswctype'),
-    iswspace: NYI('iswspace'),
-    __lctrans: NYI('__lctrans'),
-    __lctrans_cur: NYI('__lctrans_cur'),
+    __floatunsitf: NYI('__floatunsitf'),
+    __getf2: NYI('__getf2'),
+    getrlimit: NYI('getrlimit'),
+    __gttf2: NYI('__gttf2'),
+    ioctl: NYI('ioctl'),
+    __letf2: NYI('__letf2'),
     __lock: NYI('__lock'),
-    __lockfile: NYI('__lockfile'),
+    longjmp: NYI('longjmp'),
```

Comment:
I expect longjmp to get lowered. Same for setjmp later. We shouldn't see them here?